bin/main

```diff
@@ -1,52 +1,79 @@
 #!/usr/bin/env ruby
+require_relative '../lib/board'
+require_relative '../lib/player'
 
-posible_wins = [[1, 2, 3], [4, 5, 6], [7, 8, 9]]
-player_one_choices = []
-player_two_choices = []
-game_on = true
+# determine if an array includes given elements.
+class Array
+  def contain_all?(arr)
+    arr.all? { |elem| include?(elem) }
+  end
+end
 
-puts 'Player 1 name: '
-player_one = gets.chomp
+# PlayGame module holding the game possible wins.
+module PlayGame
+  POSSIBLE_WINS = [
+    [1, 2, 3], [4, 5, 6],
+    [7, 8, 9], [1, 5, 9],
+    [2, 5, 8], [3, 5, 7],
+    [1, 4, 7], [3, 6, 9]
+  ].freeze
```

Comment:
- [ ]  Also move this into the lib folder on a file, as no logic should be in bin/main, in addition rather than freeze the constant you can move it into a private method on a class.